docs/source/notebooks/08_Getting_and_bias_correcting_CMIP6_data.ipynb

```diff
@@ -0,0 +1,1788 @@
+{
```

Comment:
<p>There is a strong annual cycle in the corrected dataset that isn't at all in the simulations. That's expected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>The corrected version looks A LOT more like the ERA5 reference data, so I would say the method is correcting for this, yes. Now it remains to be seen if the ERA5 data is actually correct. I know @sebastienlanglois was looking into some possible artifacts. </p>

Reply:
<p>Good for now. </p>

Reply:
@richardarsenault  Indeed, please look into https://github.com/Ouranosinc/raven/issues/396#issuecomment-1140624233 for further info!

Reply:
Also, I'm not a reviewer but was wondering why .to_dataset_dict() is not used after querying the esm catalog instead of accessing files with get_mapper ? This would remove the need to instantiate a GCS File System.